bedrock/newsletter/templates/newsletter/includes/form.html

```diff
@@ -8,10 +8,21 @@
 
 {% if not success %}
   <form id="newsletter-form" class="mzp-c-newsletter-form" action="{{ url('newsletter.subscribe') }}" method="post">
-    {{ form.newsletters|safe }}
+    {% if is_multi_newsletter_form %}
+      <div id="newsletter-error-strings" data-form-checkboxes-error="{{ ftl('multi-newsletter-form-checkboxes-error') }}"></div>
+    {% endif %}
+
+    {% if not is_multi_newsletter_form %}
```

Comment:
This can be combined with the above `{% if ... %}` with an `{% else %}` here it look like.

Reply:
good catch, thanks